### DISCUSSION TITLE
the-world-as-I-see-it/

### DISCUSSION THREAD

**Discussion by giscus** [Philo-Li/philo-li.github.io#4] [General]
# the-world-as-I-see-it/

爱因斯坦有一本书《我的世界观》，其中收录了一系列爱因斯坦发表的书信文章还有公开演讲，其中一篇就是和书名同名的文章。我也想写一篇《我的世界观》。 写这篇文章的目的一是为了整理我当前的一些想法，二是想记录下那些对我来说无比重要的东西，用来自省、自律，用来在低谷的时候提醒自己，到底是什么样的力量支撑着我努力前行，什么是我的道，希望自己不要迷失方向，希望自己保持勇气和好奇。

https://www.philoli.com/the-world-as-I-see-it/

**Comment by KassadinL** [comment]
精神的独立、自由，赞~

**Comment by jiechen257** [comment]
像是 INFJ 的样子；赞～

**Comment by bTorna** [comment]
按自己的语言风格说话就暴露特征。是。我（支持）。科学science和民主democracy的信念不止…而且，也保障了我们的人性、humanity。往前走它危险。以下的内容不是AI生成。
不知从何说起。写简体中文字的人，不避免地就是许多意义上的中国人。我在反方向走了很远的路。世间之恶、之丑，强盗、骗子、奴才、和软弱的废物。我找不到一个稳定可靠的世界观。我不敢有观点、胆敢有感受。深深深深去理解大多数人。居住在中国的不同的城市来观察居民们的生活。
这账号是非国家安全级的普通匿名。是说普通人很难查到它可信的关联。它并没有用严格的假身份来注册和使用。
什么能让我活下去。最近几个月，我听音乐会。似乎一切都很糟。像是任何语言都可以怀疑都可以亵渎的。听众也很多都是很糟糕的人。不过那种许多人一起听音乐的状态，像是世界还没有那么糟。
另有很多事，当时很开心，事后更糟糕。比如饮酒，沉迷进入某个电子游戏或类似的人造虚拟的世界。
坏人之坏、恶人之恶、丑陋人之丑陋，但凡去想都伤害自己。让自己变坏变恶毒变丑。可我自己又是什么呢。哈，我也就是一个迷失方向的人。

**Comment by jschwinger233** [comment]
写得真好！棒！
